<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculadora</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(222, 239, 231);</string>
  </property>
  <widget class="QLineEdit" name="num1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hack Nerd Font Mono</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;
border: 1px solid;</string>
   </property>
   <property name="placeholderText">
    <string>ingrese un numero</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="num2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>90</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hack Nerd Font Mono</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;
border: 1px solid;</string>
   </property>
   <property name="placeholderText">
    <string>ingrese un numero</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hack Nerd Font Mono</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>RESULTADO</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbresul">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hack Nerd Font Mono</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnsumar">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>230</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hack Nerd Font Mono</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>SUMAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="return1">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hack Nerd Font Mono</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>REGRESAR</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
